### PR TITLE
Add description for alwaysUsePipeline option to user guide

### PR DESCRIPTION
**Pull request by nghialv**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/content/en/docs/user-guide/configuration-reference.md [hunk 1]
```diff
@@ -171,6 +176,12 @@ One of `yamlField` or `regex` is required.
 | outFilename | string | The filename for the decrypted secret. Empty means the same name with the sealed secret file. | No |
 | outDir | string | The directory name where to put the decrypted secret. Empty means the same directory with the sealed secret file. | No |
 
+## DeploymentPlanner
+
+| Field | Type | Description | Required |
+|-|-|-|-|
+| alwaysUsePipeline | bool | Always use the specified pipeline for all deployments. Default is `false`. | No |
```
Comment (khanhtc1202) [inline docs/content/en/docs/user-guide/configuration-reference.md:183]:
```suggestion
| alwaysUsePipeline | bool | Always use the defined pipeline to deploy the application in all deployments. Default is `false`. | No |
```
nits

Comment (nghialv) [reply docs/content/en/docs/user-guide/configuration-reference.md:183]: 👍

**Comment by khanhtc1202** [comment]
Way to go
/approve

**Pull request merged**